### PR TITLE
feat: add "Request quote" button for enterprise support on eligible apps

### PR DESCRIPTION
**Pull request by edward-ly**
One of two requirements for a new payment system on the app store (see [here](https://cloud.nextcloud.com/f/10131205) for details).

Text and icons are still temporary and can be adjusted as needed.

The link for the request quote button is also currently set to `https://nextcloud.com/get-a-quote/?utm_source=apps.nextcloud.com&utm_campaign=<app_id>`, but this is also pending confirmation from the marketing team.

Screenshot 2024-08-08 at 16-31-20 Enterprise App Support - App Store - Nextcloud

| | |
| - | - |
| Screenshot 2024-08-08 at 16-31-57 News - Apps - App Store - Nextcloud | Screenshot 2024-08-08 at 16-50-57 Select App to change Nextcloud App Store admin |

### CODE REVIEW

**Comment by DaphneMuller** [comment]
UI looks good to me. I'm checking with Frank

File: nextcloudappstore/user/views.py [hunk 1]
```diff
@@ -44,6 +44,22 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class EnterpriseAppsView(LoginRequiredMixin, TemplateView):
+    template_name = "user/enterprise-apps.html"
+
+    def post(self, request, pk):
+        app = get_object_or_404(App, pk=pk, owner=self.request.user)
+        app.is_enterprise_supported = not app.is_enterprise_supported
```
Comment (bigcat88) [inline nextcloudappstore/user/views.py:52]: may I ask why there is `not` here?

**Review by bigcat88 — APPROVED** [review]
code looks fine

**Comment by DaphneMuller** [comment]
Frank approved the UI

**Comment by edward-ly** [comment]
Thanks, will merge after confirmation from Peter.

**Pull request merged**